stories/molecules/Toast.stories.tsx

```diff
@@ -0,0 +1,38 @@
+import * as React from 'react';
+import type { Meta, StoryObj } from '@storybook/react';
+import { Toast } from '@textkernel/oneui';
+
+const meta: Meta<typeof Toast> = {
+    title: 'Molecules/Toast',
+    component: Toast,
+};
+
+export default meta;
+
+type Story = StoryObj<typeof Toast>;
+
+export const NotActionableToast: Story = {
+    name: 'Normal Toast',
+    args: {
+        title: 'Success',
+        content: 'This is a toast',
+        context: 'success',
```

Comment:
If we can use `CONTEXT[1]` it would be better, so that if it changes in the future we don't have to change it manually here